docs/modules/ref-tssc-workflow-components.adoc

```diff
@@ -148,7 +148,7 @@ Detect new, changed, or merged,  Merge Request to Release Branch
 * Application Language Packaging Tool
 
 | Static Code Analysis
-| The pipeline will perform static analysis on source code to catch defects, vulnerabilities, programatic and stylistic problems as early in the development lifecycle as possible, prior to deploying and running the code.
+| The pipeline will perform static analysis on source code to catch defects, vulnerabilities, programmatic and stylistic problems as early in the development life cycle as possible, prior to deploying and running the code.
```

Comment:
Just an opinion:   Maybe make this two sentences?   (I did not care for  'running the code'...)

The pipeline will perform static analysis on source code to identify defects, vulnerabilities, programmatic and stylistic problems as early in the development life cycle as possible.  For example, static analysis is completed prior to building, scanning and deploying the image.

Reply:
Updates made. Much better thanks.